src/components/InstanceDialog.tsx

```
@@ -27,242 +25,140 @@ export function InstanceDialog({
+              No instance information available to display.
+            </p>
+          </div>
+        </DialogContent>
+      </Dialog>
+    );
+  }
+
   return (
     <Dialog open={open} onOpenChange={onOpenChange}>
-      <DialogContent className="max-w-4xl max-h-[90vh] overflow-y-auto bg-card text-card-foreground border border-border shadow-lg rounded-xl left-[calc(50%+8rem)] translate-x-[-50%]">
+      <DialogContent className="overflow-y-auto max-w-2xl max-h-[90vh]">
```

ESLint

[error] Replace `overflow-y-auto·max-w-2xl·max-h-[90vh]` with `max-h-[90vh]·max-w-2xl·overflow-y-auto`

(prettier/prettier)